es/_posts/2025-09-05-058.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+pretitle: Creando un
+title: commit
+command: git commit
+descriptors:
+  - command: Comando para crear un commit
```

Comment:
```suggestion
  - command: Comando para crear\nun commit
```
The suggestion above should fix the issue from the image

<img width="351" height="127" alt="Image" src="https://github.com/user-attachments/assets/3a7ce7a2-798d-4c03-bafb-3b4bc0c0b4e2" />